.github/workflows/smoke-test.yml

```diff
@@ -23,23 +23,28 @@ jobs:
         run: |
           curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
           curl -Lo kind https://github.com/kubernetes-sigs/kind/releases/download/v0.18.0/kind-linux-amd64 && chmod +x kind && sudo mv kind /usr/local/bin/
+
       - name: Set up Docker QEMU
         uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3
+
       - name: Login to Container Registry
         uses: docker/login-action@e92390c5fb421da1463c202d546fed0ec5c39f20 # v3
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_TOKEN }}
+
       - name: Checkout HiveMQ Helm Charts
         uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
         with:
           path: helm-charts
+
       - name: Checkout HiveMQ Platform Operator on the default branch
         uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
         with:
           repository: hivemq/hivemq-platform-operator
           token: ${{ secrets.JENKINS_TOKEN }}
           path: hivemq-platform-operator
+
       - name: Checkout HiveMQ Platform Operator on the same branch
```

Comment:
I think we are missing the step here to checkout the HiveMQ Platform Operator on the target branch as well.

Reply:
Good catch! It's also needed for the legacy operator job in the same file.